web_app/db/crud.py

```diff
@@ -360,4 +376,38 @@ def get_total_amounts_for_open_positions(self) -> float | None:
                 return total_opened_amount
             except SQLAlchemyError as e:
                 logger.error(f"Error calculating total amount for open positions: {e}")
-                return None
\ No newline at end of file
+                return None
+
+
+class AirDropDBConnector(DBConnector):
+    """
+    Provides database connection and operations management for the AirDrop model.
+    """
+
+    def save_claim_data(self, airdrop_id: uuid.UUID, amount: Decimal) -> None:
+        """
+        Updates the AirDrop instance with claim data.
+        :param airdrop_id: uuid.UUID
+        :param amount: Decimal
+        """
+        airdrop = self.get_object(AirDrop, airdrop_id)
+        if airdrop:
+            airdrop.amount = amount
+            airdrop.is_claimed = True
+            airdrop.claimed_at = datetime.now()
+            self.write_to_db(airdrop)
+        else:
+            logger.error(f"AirDrop with ID {airdrop_id} not found")
+
+    def get_all_unclaimed(self) -> List[AirDrop]:
+        """
+        Returns all AirDrop instances where is_claimed=False.
+        :return: List[AirDrop]
+        """
+        with self.Session() as db:
+            try:
+                unclaimed = db.query(AirDrop).filter(AirDrop.is_claimed == False).all()
```

Comment:
please, be more specific, unclaimed what? I'd do like `unclaimed_records` or `unclaimed_instances` 
use `filter_by` instead `filter`